db/database_test.go

```diff
@@ -548,7 +548,8 @@ func TestDeltaSyncWhenFromRevIsChannelRemoval(t *testing.T) {
 	require.NoError(t, err, "Error creating user")
 
 	db.user = user
-	db.DbStats.InitDeltaSyncStats()
+	err = db.DbStats.InitDeltaSyncStats()
+	require.NoError(t, err)
```

Comment:
For this and the below ones can be simplified. `requre.NoError(t, db.DbStats.InitDeltaSyncStats())`